CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### 🐞 Fixed
 - Improve the stability of the reconnection process [#3303](https://github.com/GetStream/stream-chat-swift/pull/3303)
 - Fix invalid token errors considered as recoverable errors [#3303](https://github.com/GetStream/stream-chat-swift/pull/3303)
+- Fix a rare issue with incorrect message order when sending multiple messages while offline [#3311](https://github.com/GetStream/stream-chat-swift/issues/3311)
```

Comment:
This would go to the next release, so the entry would have to be moved from here.